src/components/Perils/PerilModal/PerilModal.tsx

```diff
@@ -0,0 +1,269 @@
+import styled from '@emotion/styled'
+import { colorsV3, fonts } from '@hedviginsurance/brand'
+
+import { LAPTOP_BP_UP, TABLET_BP_UP } from 'components/blockHelpers'
+import { Chevron } from 'components/icons/Chevron'
+import { Cross } from 'components/icons/Cross'
+import { Minus } from 'components/icons/Minus'
+import { Modal, ModalProps } from 'components/Modal/Modal'
```

Comment:
I think if we want to use `framer-motion` in the modal, we should code split that.